pkg/genericapiserver/config.go

```diff
@@ -282,6 +284,7 @@ func (c *Config) ApplyOptions(options *options.ServerRunOptions) *Config {
 				},
 			},
 			SNICerts: []NamedCertKey{},
+			ServerCA: options.TLSCAFile,
```

Comment:
doc for `--tls-cert-file` already expects the CA to be concatenated to the cert file:

> --tls-cert-file string
> File containing x509 Certificate for HTTPS. (CA cert, if any, concatenated after server cert)